packages/editor/src/components/inserter-sidebar/index.js

```diff
@@ -20,45 +17,48 @@ export default function InserterSidebar( {
 	closeGeneralSidebar,
 	isRightSidebarOpen,
 } ) {
-	const { insertionPoint, showMostUsedBlocks, blockSectionRootClientId } =
+	const { inserterSidebarToggleRef, insertionPoint, showMostUsedBlocks } =
 		useSelect( ( select ) => {
-			const { getInsertionPoint } = unlock( select( editorStore ) );
-			const {
-				getBlockRootClientId,
-				__unstableGetEditorMode,
-				getSettings,
-			} = select( blockEditorStore );
+			const { getInserterSidebarToggleRef, getInsertionPoint } = unlock(
+				select( editorStore )
+			);
 			const { get } = select( preferencesStore );
-			const getBlockSectionRootClientId = () => {
```

Comment:
Is this removal intentional? It seems it was added just yesterday [here](https://github.com/WordPress/gutenberg/pull/61464).

Reply:
I bet it was accidentally removed during a rebase. Nice catch!